src/controller/CHIPDeviceController.cpp

```diff
@@ -142,6 +142,10 @@ CHIP_ERROR DeviceController::Init(ControllerInitParams params)
 
     mSystemState = params.systemState->Retain();
     mState       = State::Initialized;
+
+    ChipLogProgress(Controller, "Joined the fabric at index %d. Compressed fabric ID is: 0x" ChipLogFormatX64, GetFabricIndex(),
```

Comment:
This log should only happen in the case when we called `InitControllerNOCChain`.  Because otherwise we have not in fact joined a fabric.

Reply:
I've wrapped it with a conditional check on a valid fabric index being present.